tv-shows/src/components/features/review/ReviewItem.tsx

```diff
@@ -2,13 +2,24 @@ import { Text, Card, CardBody } from "@chakra-ui/react";
 import { CloseIcon } from "@chakra-ui/icons";
 import { IReview } from "@/typings/Review.type";
 import { StarReviewIcons } from "@/components/shared/stars/StarReviewIcons";
+import { useParams } from "next/navigation";
+import { useSWRConfig } from "swr";
+import { swrKeys } from "@/fetchers/swrKeys";
+import { deleteReview } from "@/fetchers/mutators";
 
 interface IReviewItemProps {
   review: IReview;
-  onDelete: (review: IReview) => void;
 }
 
-export const ReviewItem = ({ review, onDelete }: IReviewItemProps) => {
+export const ReviewItem = ({ review }: IReviewItemProps) => {
+  const params = useParams();
+  const { mutate } = useSWRConfig();
+
+  const onDelete = (reviewId: string) => {
+    deleteReview(swrKeys.reviewItem(reviewId));
```

Comment:
S ovim ideš direktno na API (zaobilaziš SWR). 
Točnije je da koristimo useSWRMutation i radimo trigger s ovom akcijom, a onda u onSuccess napravimo ažuriranje cache-a .